sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetScan.scala

```diff
@@ -944,30 +945,30 @@ private case class GpuParquetFileFilterHandler(
    */
   @scala.annotation.nowarn("msg=method getDecimalMetadata in class PrimitiveType is deprecated")
   private def checkPrimitiveCompat(pt: PrimitiveType,
-                                   dt: DataType,
-                                   errorCallback: () => Unit): Unit = {
```

Comment:
nit: most of the changes to this file are formatting and have nothing to do with the changes being made. Can we please revert these so there is less of a chance of code conflicts?